### PR TITLE
Use inline variable expansion for wait_boot parameter

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by coolo** [comment]
This makes it quite hard to read. Perhaps you can inject an helper function?

**Comment by okurz** [comment]
really? you mean like
```perl
sub is_minimal_system_applicable {
    get_var("DESKTOP") =~ /textmode|minimalx/
}
```

**Comment by coolo** [comment]
is_desktop_installed would do for me

**Comment by okurz** [comment]
like this? I did not conduct a full online migration test just a quick startup of "does isotovideo fail immediately?". So if you trust this enough and like it, merge it

File: lib/utils.pm [hunk 1]
```diff
@@ -367,6 +368,10 @@ sub ensure_fullscreen {
     }
 }
 
+sub is_desktop_installed {
+    return get_var("DESKTOP") =~ /textmode|minimalx/;
```
Comment (coolo) [inline lib/utils.pm]: that is the opposite of what you want to achieve

**Comment by coolo** [comment]
beside the bug - yes, like that

**Comment by okurz** [comment]
updated

**Comment by okurz** [comment]
rebased

**Pull request merged**